From 26e6c07e24b9d2f19a3362a90902590a4e0f9e48 Mon Sep 17 00:00:00 2001
From: Julian Lehrhuber <jl@lerry.de>
Date: Mon, 10 Feb 2020 09:31:15 +0100
Subject: [PATCH 2/3] Disable default folder creation

---
 cmd/syncthing/main.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmd/syncthing/main.go b/cmd/syncthing/main.go
index 37872fcf..35366377 100644
--- a/cmd/syncthing/main.go
+++ b/cmd/syncthing/main.go
@@ -137,7 +137,7 @@ The following are valid values for the STTRACE variable:
 // Environment options
 var (
 	innerProcess    = os.Getenv("STNORESTART") != "" || os.Getenv("STMONITORED") != ""
-	noDefaultFolder = os.Getenv("STNODEFAULTFOLDER") != ""
+	noDefaultFolder = true
 )
 
 type RuntimeOptions struct {
-- 
2.18.0.windows.1

